pages/wallet.js

```diff
@@ -127,7 +127,46 @@ export function FundForm () {
         />
         <SubmitButton variant='success' className='mt-2'>generate invoice</SubmitButton>
       </Form>
-      <WalletHistory />
+    </>
+  )
+}
+
+export function WithdrawalMethods () {
+  const router = useRouter()
+
+  return (
+    <>
+      <YouHaveSats />
+      <Tabs
+        defaultActiveKey={
+          router.query.type === 'lnurl-withdraw'
+            ? 'qrcode'
+            : router.query.type === 'lnaddr-withdraw'
+              ? 'lnaddr'
+              : 'invoice'
+        }
+        id='withdrawal-tabs'
+        justify
+        mountOnEnter
+        onSelect={(k, e) => {
+          router.replace(
```

Comment:
Nit: This might be better as `router.push(..., ..., {shallow: true})` so that back/forward navigation can be used.